### ISSUE TITLE
warmup sharing

### ISSUE THREAD

**Issue by hyunjimoon** [hyunjimoon/SBC#20]
1. Increase the number of num_warmup from M to at least 500
2. Only warm-up for a large number of samples at first. For the next step (N=2..), we could borrow the previous learned parameters
```
 sc_workflow$fit_model(sample_iterations = M, warmup_iterations = 500, data)
```

Seems like we can use `metric_file` as suggested [here](https://discourse.mc-stan.org/t/saved-mass-matrix/2921/17?u=hyunji.moon)

**Comment by martinmodrak** [comment]
I don't think that would work that well - different datasets could require very different warmups, so we can at best share warmup between chains for the same fit. And this is probably best done within the backend (e.g. in Stan)
